FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta2/c_sdk/platform/freertos/transport/src/tls_freertos.c

```diff
@@ -576,29 +583,29 @@ int32_t TLS_FreeRTOS_send( NetworkContext_t * pNetworkContext,
                            const void * pBuffer,
                            size_t bytesToSend )
 {
-    int32_t socketStatus = 0;
+    int32_t tlsStatus = 0;
 
-    socketStatus = ( int32_t ) mbedtls_ssl_write( &( pNetworkContext->sslContext.context ),
-                                                  pBuffer,
-                                                  bytesToSend );
+    tlsStatus = ( int32_t ) mbedtls_ssl_write( &( pNetworkContext->sslContext.context ),
+                                               pBuffer,
+                                               bytesToSend );
 
-    if( ( socketStatus == MBEDTLS_ERR_SSL_TIMEOUT ) ||
-        ( socketStatus == MBEDTLS_ERR_SSL_WANT_READ ) ||
-        ( socketStatus == MBEDTLS_ERR_SSL_WANT_WRITE ) )
+    if( ( tlsStatus == MBEDTLS_ERR_SSL_TIMEOUT ) ||
+        ( tlsStatus == MBEDTLS_ERR_SSL_WANT_READ ) ||
+        ( tlsStatus == MBEDTLS_ERR_SSL_WANT_WRITE ) )
     {
         /* Mark these set of errors as a timeout for the libraries to retry
          * the send. */
-        socketStatus = 0;
+        tlsStatus = 0;
     }
-    else if( socketStatus < 0 )
+    else if( tlsStatus < 0 )
     {
-        LogError( ( "Error %d while sending data.", socketStatus ) );
+        LogError( ( "Error %d while sending data.", tlsStatus ) );
```

Comment:
Should these return values of read and write functions be logged as strings?
```suggestion
        LogError( ( "Failed to send data: ErrorStatus=%d", tlsStatus ) );
```